CFC_DataCollector/recommender/alternative_trips_pipeline.py

```diff
@@ -0,0 +1,56 @@
+import get_trips
+from common import store_trip_in_db
+#import get_trips
+from trip import E_Mission_Trip
+#import Profiles
+
+def alternative_trips(trip_list):
```

Comment:
what is the difference between this method and `def get_alternative_trips`? And why is the implementation just storing information and not returning anything?